addons/storyshots/storyshots-core/src/frameworks/angular/loader.ts

```diff
@@ -42,7 +44,9 @@ function load(options) {
   };
 }
 
-export default {
+const angularLoader: Loader = {
```

Comment:
It's the way I found to have type-check according to `Loader` class, if anyone has a better proposition?